Makefile

```diff
@@ -70,6 +70,10 @@ generate: generate-testdata generate-docs ## Update/generate all mock data. You
 generate-testdata: ## Update/generate the testdata in $GOPATH/src/sigs.k8s.io/kubebuilder
 	chmod -R +w testdata/
 	rm -rf testdata/
+	chmod -R +w docs/book/src/cronjob-tutorial/testdata/project/bin/
+	chmod -R +w docs/book/src/cronjob-tutorial/testdata/project/bin/
+	rm -rf docs/book/src/component-config-tutorial/testdata/project/bin/
+	rm -rf docs/book/src/cronjob-tutorial/testdata/project/bin/
```

Comment:
We remove the docs testdata via the [hack/](https://github.com/kubernetes-sigs/kubebuilder/blob/master/hack/docs/ scripts already

So, if you would like to add the chmod I think would be better set in: https://github.com/kubernetes-sigs/kubebuilder/blob/master/hack/docs/

Reply:
OK

Reply:
> We remove the docs testdata via the [hack/](https://github.com/kubernetes-sigs/kubebuilder/blob/master/hack/docs/ scripts already

Hi, @camilamacedo86 , I didn't find the relative code removing docs/testdata. Could you please point out which files/lines? Thanks very much.

Reply:
I find it in Prepare() function.